core-js/src/main/javascript/cdf/components/filter/views/scrollbar/MCustomScrollBarEngineImplementation.js

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright 2002 - 2016 Webdetails, a Pentaho company. All rights reserved.
+ *
+ * This software was developed by Webdetails and is provided under the terms
+ * of the Mozilla Public License, Version 2.0, or any later version. You may not use
+ * this file except in compliance with the license. If you need a copy of the license,
+ * please go to http://mozilla.org/MPL/2.0/. The Initial Developer is Webdetails.
+ *
+ * Software distributed under the Mozilla Public License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. Please refer to
+ * the license for the specific language governing your rights and limitations.
+ */
+/**
+ * @summary MCustomScrollBarEngineImplementation implementation of ScrollBarHandler
+ * @description MCustomScrollBarEngineImplementation implementation of ScrollBarHandler
+ */
+define([
+  './AbstractScrollBarHandler',
+  '../../../../lib/jquery',
+  'amd!../../../../lib/jquery.mCustomScrollbar'
+],function(AbstractScrollBarHandler, $){
+  return AbstractScrollBarHandler.extend({
```

Comment:
This implementation is mostly correct, I don't understand why you chose a different implementation in the legacy version.